<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showWidget</class>
 <widget class="QWidget" name="showWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑 Light</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>遍历显示窗口</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>681</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string>学号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>性别</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>生日</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电话</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>住址</string>
    </property>
   </column>
  </widget>
  <widget class="QRadioButton" name="radioButtonPreOrder">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>188</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>前序遍历</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonInOrder">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>中序遍历</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonPostOrder">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>后序遍历</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>130</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>遍历方式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSearch">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>290</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonYoungestSearch">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>390</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>显示年龄最小学生信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
